Source/Fuse.Navigation/Router.ScriptClass.uno

```diff
@@ -175,21 +175,21 @@ namespace Fuse.Navigation
 				- `relative`: An optional node that indicates the path is relative to this router outlet. The path is specified like in `gotoRelative`
 				- `style`: The style of the operation, which can be used as a matching criteria in transitions.
 		*/
-		static void Modify(Context c, Router r, object[] args)
+		static object Modify(Context c, Router r, object[] args)
```

Comment:
What is the significance of the `object` return? Do signatures with `Context` still run on the JS thread?

Reply:
Why does this function take a `Context` but `NavigationControl.modifyPath` not take a `Context`? Is it required here only to the `Wrap` call on the nodes?  Do we even need `Wrap` anymore, or does reflection take care of it.

Reply:
object is just a return-value, and is only available on things that run on the JS-thread (same as with Context).

The reason why this takes a Context is that ScriptRouterRequest takes a Context. Perhaps it wouldn't have to? If so, we could get rid of the closure.

Reply:
It was using the context to call `Wrap` for the `Node`. But if you're saying that was a no-op anyway, chances are we don't need this context here, and we don't need this function to run on the JS thread at all.

Reply:
Yeah. I've updated this in this PR already, and that made the method a lot more pleasant to read.